.ci/build-mac.sh

```diff
@@ -17,10 +17,14 @@ cp target/release/libportability.dylib vulkan-sdk/lib/libVulkan.dylib
 install_name_tool -id "${PWD}"/vulkan-sdk/lib/libVulkan.dylib vulkan-sdk/lib/libVulkan.dylib
 export VULKAN_SDK="${PWD}/vulkan-sdk"
 
+<<<<<<< HEAD:.ci/build-mac.sh
```

Comment:
Messed with rebase?

Reply:
Always happens when I use Atom to fix those, will manually correct later